make_aos_vs_piff_graph.py

```diff
@@ -0,0 +1,353 @@
+
+    plt.figure()
+    plt.scatter(aos_z6y_list, piff_z6y_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z6y")
+    plt.savefig("{0}/aos_vs_piff_z6y_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z7d_list, piff_z7d_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z7d")
+    plt.savefig("{0}/aos_vs_piff_z7d_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z7x_list, piff_z7x_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z7x")
+    plt.savefig("{0}/aos_vs_piff_z7x_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z7y_list, piff_z7y_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z7y")
+    plt.savefig("{0}/aos_vs_piff_z7y_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z8d_list, piff_z8d_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z8d")
+    plt.savefig("{0}/aos_vs_piff_z8d_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z8x_list, piff_z8x_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z8x")
+    plt.savefig("{0}/aos_vs_piff_z8x_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z8y_list, piff_z8y_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z8y")
+    plt.savefig("{0}/aos_vs_piff_z8y_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z9d_list, piff_z9d_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z9d")
+    plt.savefig("{0}/aos_vs_piff_z9d_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z10d_list, piff_z10d_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z10d")
+    plt.savefig("{0}/aos_vs_piff_z10d_{1}.png".format(graph_directory, psf_type))
+
+    plt.figure()
+    plt.scatter(aos_z11d_list, piff_z11d_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("z11d")
+    plt.savefig("{0}/aos_vs_piff_z11d_{1}.png".format(graph_directory, psf_type))
+
+    try:
+        plt.figure()
+        plt.scatter(aos_z14d_list, piff_z14d_list)
+        plt.xlabel("aos")
+        plt.ylabel("piff")
+        plt.title("z14d")
+        plt.savefig("{0}/aos_vs_piff_z14d_{1}.png".format(graph_directory, psf_type))
+
+        plt.figure()
+        plt.scatter(aos_z15d_list, piff_z15d_list)
+        plt.xlabel("aos")
+        plt.ylabel("piff")
+        plt.title("z15d")
+        plt.savefig("{0}/aos_vs_piff_z15d_{1}.png".format(graph_directory, psf_type))
+    except:
+	pass
+
+    plt.figure()
+    plt.scatter(aos_r0_list, piff_opt_size_list)
+    plt.xlabel("aos")
+    plt.ylabel("piff")
+    plt.title("aos r0 vs piff opt size")
+    plt.savefig("{0}/aos_r0_vs_piff_opt_size_{1}.png".format(graph_directory, psf_type))
+
+
+
+if __name__ == '__main__':
```

Comment:
this whole file could have been probably 30 lines if you employed for loops! what if you decide (e.g.) that you want your xlabel to say "Active Optics System" and need to modify the title accordingly?